faker/providers/person/bs_BA/__init__.py

```diff
@@ -0,0 +1,88 @@
+# coding=utf-8
+from __future__ import unicode_literals
+from .. import Provider as PersonProvider
+
+
+class Provider(PersonProvider):
+    """
+    The names used here are the top 100 names released by the Federal statistics 
```

Comment:
there are some trailing spaces at the end of hits and the next line. Can you remove them?